app/src/sharedTest/java/org/oppia/app/player/state/StateFragmentPresenterTest.kt

```diff
@@ -0,0 +1,81 @@
+package org.oppia.app.player.state;
+
+import android.provider.MediaStore
+import android.view.View
+import androidx.test.core.app.ActivityScenario
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.matcher.ViewMatchers
+import androidx.test.espresso.matcher.ViewMatchers.isDescendantOfA
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.espresso.matcher.ViewMatchers.withText
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.hamcrest.Matchers.allOf
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.oppia.app.R
+import org.oppia.app.home.HomeActivity
+import org.oppia.app.player.exploration.ExplorationActivity
+import android.widget.TextView
+import android.widget.LinearLayout
+import android.widget.RadioGroup
+import androidx.test.espresso.matcher.ViewMatchers.isDisplayed
+import androidx.test.espresso.ViewAction
+import androidx.test.platform.ui.UiController
+import junit.framework.TestCase.assertEquals
+import org.hamcrest.Matcher
+
+/** Tests for [StateFragmentPresenter]. */
+@RunWith(AndroidJUnit4::class)
+class StateFragmentPresenterTest {
+  var interactionInstanceId: String? = null
+
+  @Test
+  fun testMultipleSelectionInputInteraction() {
+    ActivityScenario.launch(ExplorationActivity::class.java).use {
+      interactionInstanceId = "MultipleChoiceInput"
+      assertEquals(interactionInstanceId, "MultipleChoiceInput")
+      onView(withId(R.id.interactionRadioGroup)).perform(object : ViewAction {
```

Comment:
Could we use the built-in click() view action if we used a recycler view and something like: https://developer.android.com/training/testing/espresso/lists?

I think we should avoid custom ViewActions unless there isn't a good alternative.

Reply:
Used built-in click() of Recyclerview `onView(withId(R.id.rvInteractions)).perform(
        RecyclerViewActions.actionOnItemAtPosition<RecyclerView.ViewHolder>(2,click()))`